Dockerfile

```diff
@@ -24,35 +24,78 @@ RUN curl --location https://github.com/marbl/Mash/releases/download/v2.2/mash-Li
     rm -r mash*
 
 
-RUN apt-get install --yes gcc && \
-    apt-get install --yes --reinstall zlibc zlib1g zlib1g-dev
+RUN apt-get install --yes gcc=4:6.3.0-4 && \
+    apt-get install --yes --reinstall zlibc=0.9k-4.3 zlib1g=1:1.2.8.dfsg-5 zlib1g-dev=1:1.2.8.dfsg-5
 
+# dRep says v2.6.3 (2016) confirmed works, on its wiki
 RUN git clone https://github.com/hyattpd/Prodigal && \
     cd Prodigal/ && \
+    git checkout a78ed3f46f46dd51d6fca47a8b49e809342e45cd && \
     make install && \
     cd .. && \
     rm -rf Prodigal
 
 
 RUN curl --location https://github.com/matsen/pplacer/releases/download/v1.1.alpha17/pplacer-Linux-v1.1.alpha17.zip > pplacer.zip && \
     unzip pplacer.zip && \
-    cd pplacer-Linux-v1.1.alpha17/ && \
-    mv * /usr/local/bin && \
-    cd .. && \
+    mv pplacer-Linux-v1.1.alpha17 /usr/local/bin/ && \
     rm -r pplacer*
 
+ENV PATH="${PATH}:/usr/local/bin/pplacer-Linux-v1.1.alpha17"
+
+
+RUN apt-get install --yes hmmer=3.1b2+dfsg-5
+
+
+RUN apt-get install --yes libbz2-dev=1.0.6-8.1 liblzma-dev=5.2.2-1.2+b1
+
+RUN pip install checkm-genome==1.1.1
+
+# just to get dataRoot pointing to /data/CHECKM_DATA without a 'Permission denied' error
+# does this work when re-caching Dockerfile on narrative without running refdata scripts? (yes, it will create /data/CHECKM_DATA)
+RUN checkm data setRoot /data/CHECKM_DATA && \
+    echo "ls -a /data/CHECKM_DATA" && ls -a /data/CHECKM_DATA && \
+    echo "cat /miniconda/lib/python3.6/site-packages/checkm/DATA_CONFIG" && cat /miniconda/lib/python3.6/site-packages/checkm/DATA_CONFIG
+
+
+RUN apt-get install g++=4:6.3.0-4 csh=20110502-2.2+b1 --yes
+
+RUN curl --location https://sourceforge.net/projects/mummer/files/mummer/3.23/MUMmer3.23.tar.gz/download > mummer3.23.tar.gz && \
+tar -vxzf mummer3.23.tar.gz  && \
```

Comment:
Indent plz, as well as line 74

Reply:
And 82

Reply:
why should i indent this? i never felt that would increase the readability or robustness of the code, just make for more typing and fuss